lib/validate.js

```diff
@@ -41,6 +41,20 @@ async function validateMetadata(descriptor) {
 
   const profile = await Profile.load(defaultProfile)
 
+  // TODO: remove this temporal fix, when
+  // TODO: https://github.com/frictionlessdata/tableschema-js/issues/133 is resolved
+  // datahub.io remote site Fails if there is  '../'  in the resource.path
+  // https://github.com/datahq/data-cli/issues/265
+  descriptor.resources.map(res => {
+    if (res.path.includes('../') || res.path.startsWith('/')){
```

Comment:
@AcckiyGerman here are all unsafe paths that `datapackage-py` is checking against. Think we should consider all of them to be aligned with them.

https://github.com/frictionlessdata/datapackage-py/blob/master/datapackage/helpers.py#L202-L209

Reply:
cc @anuveyatsu